src/routes/__layout.svelte

```diff
@@ -7,3 +7,76 @@
+  @import "./../../node_modules/@ons/design-system/components/icons/icons";
+  @import "./../../node_modules/@ons/design-system/components/images/images";
+  @import "./../../node_modules/@ons/design-system/components/input/input-type";
+  @import "./../../node_modules/@ons/design-system/components/input/input";
+  @import "./../../node_modules/@ons/design-system/components/label/label";
+  @import "./../../node_modules/@ons/design-system/components/language-selector/language";
+  @import "./../../node_modules/@ons/design-system/components/lists/list";
+  @import "./../../node_modules/@ons/design-system/components/message-list/message-list";
+  @import "./../../node_modules/@ons/design-system/components/message/message";
+  @import "./../../node_modules/@ons/design-system/components/metadata/metadata";
+  @import "./../../node_modules/@ons/design-system/components/modal/modal";
+  @import "./../../node_modules/@ons/design-system/components/pagination/pagination";
+  @import "./../../node_modules/@ons/design-system/components/panel/panel";
+  @import "./../../node_modules/@ons/design-system/components/phase-banner/phase-banner";
+  @import "./../../node_modules/@ons/design-system/components/promotional-banner/promo-banner";
+  @import "./../../node_modules/@ons/design-system/components/question/question";
+  @import "./../../node_modules/@ons/design-system/components/quote/quote";
+  @import "./../../node_modules/@ons/design-system/components/radios/radio";
+  @import "./../../node_modules/@ons/design-system/components/radios/radios";
+  @import "./../../node_modules/@ons/design-system/components/related-content/related-content";
+  @import "./../../node_modules/@ons/design-system/components/relationships/relationships";
+  @import "./../../node_modules/@ons/design-system/components/search/search";
+  @import "./../../node_modules/@ons/design-system/components/section-navigation/section-navigation";
+  @import "./../../node_modules/@ons/design-system/components/skip-to-content/skip";
+  @import "./../../node_modules/@ons/design-system/components/status/status";
+  @import "./../../node_modules/@ons/design-system/components/summary/summary";
+  @import "./../../node_modules/@ons/design-system/components/table-of-contents/toc";
+  @import "./../../node_modules/@ons/design-system/components/table/table";
+  @import "./../../node_modules/@ons/design-system/components/tabs/tabs";
+  @import "./../../node_modules/@ons/design-system/components/text-indent/text-indent";
+  @import "./../../node_modules/@ons/design-system/components/timeline/timeline";
+  @import "./../../node_modules/@ons/design-system/components/upload/upload";
+  @import "./../../node_modules/@ons/design-system/components/video/video";
+  @import "./../../node_modules/@ons/design-system/scss/utilities/index";
+  @import "./../../node_modules/@ons/design-system/scss/overrides/rtl";
+  @import "./../../node_modules/@ons/design-system/scss/overrides/hcm";
+</style>
```

Comment:
Can you please also move over the surrounding comments around that block? This whole thing can likely replaced with the following now that the `!default` bug is fixed on upstream:
```
<style lang="scss" global>
  $static: "";
  @import "./../../node_modules/@ons/design-system/scss/main.scss";
</style>
```
but we need to bump DS version to make use of the fix which means going through the components and changelog which we don't really want right now.

We should definitely drop this behemoth asap and `XXX` comments are great for pointing workarounds.

See [old repo for context](https://github.com/ONSdigital/census-atlas/blob/master/src/routes/__layout.svelte#L99).